dev/bots/test.dart

```diff
@@ -389,6 +390,14 @@ Future<void> _runExampleProjectBuildTests(FileSystemEntity exampleDirectory) asy
       print('Example project ${path.basename(examplePath)} has no ios directory, skipping ipa');
     }
   }
+  if (Platform.isMacOS && (branch != 'beta' && branch != 'stable')) {
```

Comment:
I would pull this along with the branch code about into a single "shouldRunMacOS" getter and document that it needs to stay in sync with features.dart in the flutter tool.

It would be nice the flutter tool had machine readable flutter config/features output, then we wouldn't need to keep track of this in 2 places.

Reply:
Done.